### ISSUE TITLE
TagOwners -> tagowners = error

### ISSUE THREAD

**Issue by lizdeika** [davidsbond/terraform-provider-tailscale#11]
Hi,

Access Controls TagOwners looks like this:
```json
  "TagOwners": {
    "tag:tailscale-relay": [
      "group:admins"
    ]
  },
```

But I get such an error for ACL:
```
│ Error: Failed to set ACL
│ 
│   with tailscale_acl.current_acl,
│   on main.tf line 19, in resource "tailscale_acl" "current_acl":
│   19: resource "tailscale_acl" "current_acl" {
│ 
| ports: tag not found: "tag:tailscale-relay" (400)
```

I think the problem is that all the json keys are downcased by terraform tailscale provider like `tagowners`
so tailscale can not find a key `TagOwners` with defined tags 
and fails with an error.

**Comment by davidsbond** [comment]
Hey @lizdeika, thanks for raising this issue. 

I suspect the problem is because the `TagOwners` field is currently not supported in this provider. You can see the structure of the ACL data [here](https://github.com/davidsbond/terraform-provider-tailscale/blob/master/internal/tailscale/client.go#L175). So when you make your change, it likely isn't including `TagOwners` in the request itself.

I haven't been keeping up with the Tailscale ACL features, and I don't think this was a thing when I first put this provider together. I also need to add the `Tests` field by the look of it too.

I'll happily review a pull request if you'd like to raise one. Otherwise, if you can wait, I'll take a look at implementing this once I'm free to.

**Comment by davidsbond** [comment]
Also, the way the provider is currently implemented it expects all lowercase JSON, which is supported by the Tailscale API, and it does not use their HuJSON package. So comments and trailing commas will not work currently.

**Comment by davidsbond** [comment]
Hey @lizdeika, it turned out this was fairly easy to implement. I've released a new version of the provider, version 0.1.0, which will support the `Tests` and `TagOwners` fields. I tested against the ACL for my personal setup like so:

```json
{
  "acls": [
    {
      "action": "accept",
      "ports": [
        "*:*"
      ],
      "users": [
        "davidsbond93@gmail.com"
      ]
    }
  ],
  "groups": {
    "group:david": [
      "davidsbond93@gmail.com"
    ]
  },
  "tagowners": {
    "tag:homelab": [
      "group:david"
    ]
  }
}
```

And the Tailscale API was happy to accept it. Hope this helps.

**Comment by lizdeika** [comment]
Wow! That went hyper fast :D
Thank you very much, version `0.1.0` worked like a charm!
